azure-arc-for-servers/get-kvsecrets-from-arc-servers.ps1

```diff
@@ -0,0 +1,65 @@
+# run the following on your Azure ARC machines to get secrets from you Key Vault
+function Get-AccessToken {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$true)]
+        [string] $audience,
+        [string] $apiVersion = "2019-08-15"
+    )
+
+    $azcmagent = azcmagent show
+    if (-not ($azcmagent -like "*: Connected"))
+    {
+        Write-Error "Ensure Azure ARC machine is onboarded and himds sevice is running."
+        return
+    }
+        
+    $identityEndpoint = $env:IDENTITY_ENDPOINT
+    if(-not ($identityEndpoint)) {
+        $identityEndpoint = "http://localhost:40342/metadata/identity/oauth2/token"
+    }
```

Comment:
Can `identityEndpoint` be a parameter for the `Get-AccessToken` instead of relying on environment variable?

Reply:
identityEndpoint is actually a environment variable which our ARC agent package sets it up.